.tekton/osc-caa-webhook-pull-request.yaml

```diff
@@ -29,6 +29,8 @@ spec:
     value: Dockerfile
   - name: path-context
     value: src/webhook
+  - name: prefetch-input
```

Comment:
In particular given that this is a different repository, could you put in the commit message the same indications you put in the commit message for the `sandboxed-container-operator`?

```
Following Konflux doc:
- https://konflux-ci.dev/docs/building/prefetching-dependencies/#enabling-prefetch-builds-for-rpm
- https://konflux-ci.dev/docs/building/activation-keys-subscription/#configuring-an-rpm-lockfile-for-hermetic-builds
```

Reply:
The links you mention are for RPM pre-fetching, which is not relevant for this PR.
I'm adding a link to the general pre-fetch documentation.